monitor/account/account.go

```diff
@@ -25,15 +38,15 @@ func Monitor(ctx context.Context, network netconf.Network) error {
 		rpcClientPerChain[chain.ID] = c
 	}
 
-	addresses := map[netconf.ID][]common.Address{
+	addresses := map[netconf.ID][]accountType{
 		netconf.Testnet: {
-			contracts.TestnetCreate3Deployer(),
-			contracts.TestnetDeployer(),
+			{contracts.TestnetCreate3Deployer(), create3Deployer},
+			{contracts.TestnetDeployer(), deployer},
 		},
 		netconf.Staging: {
-			contracts.StagingCreate3Deployer(),
-			contracts.StagingDeployer(),
-			common.HexToAddress("0x7a6cF389082dc698285474976d7C75CAdE08ab7e"), // fb: dev
+			{contracts.StagingCreate3Deployer(), create3Deployer},
+			{contracts.StagingDeployer(), deployer},
+			{common.HexToAddress("0x7a6cF389082dc698285474976d7C75CAdE08ab7e"), devFireblocks}, // fb: dev
```

Comment:
Do we need to monitor Relayer address as well?

Reply:
currently done by the relayer itself, we would need to supply the address to the monitor. I'll do it in next pr